<launch>
  <arg name="voxblox_path" default="/home/helen/data/jfr_2018/machine_hall/voxblox/rs_esdf_0.10.voxblox" />
  <arg name="frame_id" default="map" />

  <node name="voxblox_rrt_planner" pkg="voxblox_rrt_planner" type="voxblox_rrt_planner_node" output="screen">
    <param name="voxblox_path" value="$(arg voxblox_path)" />
    <param name="visualize" value="true" />
    <param name="robot_radius" value="0.5" />
    <param name="tsdf_voxel_size" value="0.10" />
    <param name="tsdf_voxels_per_side" value="16" />
    <param name="update_mesh_every_n_sec" value="0.0" />
    <param name="num_seconds_to_plan" value="5.0" />
    <param name="frame_id" value="$(arg frame_id)" />
    <param name="world_frame" value="$(arg frame_id)" />
    <param name="trust_approx_solution" value="true" />
    <param name="publish_traversable" value="true" />
    <param name="split_at_collisions" value="true" />
    <param name="optimize_time" value="true" />
    <param name="verbose" value="true" />
  </node>

</launch>
